doc/release/RELEASE-NOTES.md

```diff
@@ -22,6 +22,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Architecture/Environment changes
 * [OSDEV-1325](https://opensupplyhub.atlassian.net/browse/OSDEV-1325)
   * __Deploy to AWS__ pipeline will init from __[Release] Deploy__ pipeline and get deployment parameters, such as cleaning OpenSearch indexes, by trigger.
+* [OSDEV-1372](https://opensupplyhub.atlassian.net/browse/OSDEV-1372)
+  * Updating Dockerfile with newer Debian version as PostgreSQL repository support for Debian 10 has ended
```

Comment:
1. I think we need to write such updates as factual statements. For example: "Updat**ed** something-something-something."
2. Please add a trailing full stop to the sentence.
3. Also, please specify which application you updated the Dockerfile for. It is a Django app.

Reply:
Also, please do the same changes in the description of the PR

Reply:
done